types/ramda/ramda-tests.ts

```diff
@@ -2521,6 +2521,26 @@ class Rectangle {
     R.symmetricDifferenceWith(eqL)(l1, l2); // => ['dddd', 'c']
 };
 
+() => {
+    const list1 = [
+        {id: 1, name: 'One'},
+        {id: 2, name: 'Two'},
+        {id: 3, name: 'Three'},
+        {id: 4, name: 'Four'},
+        {id: 5, name: 'Five'}
+    ];
+
+    const list2 = [5, 4, 6];
+    function matchId(record: any, id: number): boolean {
```

Comment:
You could consider removing `any` and use an arrow function instead:

`const matchId = (record: { id:number, name: string}, id: number): boolean => record.id === id`